docs/lib/project-setup/fragments/android/coroutines/coroutines.md

```diff
@@ -0,0 +1,125 @@
+Amplify provides an optional and separate API surface which is entirely focused on using Kotlin's [coroutines](https://developer.android.com/kotlin/coroutines) and [flows](https://developer.android.com/kotlin/flow).
+
+To use it, import **`Amplify`** facade from `core-kotlin` instead of from `core`.
+
+With the Coroutines APIs, most Amplify functions are expressed as `suspend` functions:
+
+```kotlin
+import com.amplifyframework.kotlin.core.Amplify
+// ...
+
+val post = Post.builder()
+    .title("My First Post")
+    .build()
+
+lifecycleScope.launch {
```

Comment:
May be worth referencing some information regarding `lifecycleScope` at the top. Most folks will probably know what it is, but there's always a chance someone may just paste the code sample a wonder where that comes from. Thoughts?